ironfish/src/primitives/block.test.ts

```diff
@@ -8,10 +8,11 @@ import {
 } from '../testUtilities/fixtures'
 import { makeBlockAfter } from '../testUtilities/helpers/blockchain'
 import { createNodeTest } from '../testUtilities/nodeTest'
-import { SerializedBlock } from './block'
+import { BlockSerde, SerializedBlock } from './block'
 
 describe('Block', () => {
   const nodeTest = createNodeTest()
+  const blockSerde = BlockSerde
```

Comment:
Is there some mocking reason we can't just use `BlockSerde` everywhere in this file?

Reply:
No - relic from before I made them static. Will update